slp-postage-protocol.md

```diff
@@ -0,0 +1,268 @@
+
+The additional non-SLP BCH inputs attached to most SLP transactions are akin to the concept of "gas" as used in Ethereum. In order to send a token from one party to another, on-chain, a wallet must also have some amount of the additional native token (in this case BCH satoshis) on hand. This limitation introduces significant friction to the user experience. An SLP token sent, for instance, to a fresh wallet cannot be utilized by the recipient unless he or she is also in possession of additional BCH. This means that a sender must either send some additional BCH along with the SLP tokens, or the recipient must acquire BCH through other means - a potentially daunting task for new users.
+
+The protocol described in the following specification allows any SLP wallet to engage an intermediary (**"post office"**) who will add BCH inputs (**"postage"**) to the sender's transaction, and then broadcast it to the network. In exchange for providing this service, the post office can require payment, in the form of the SLP token, as an additional output in the transaction. This interaction effectively allows a sender to pay his network fees in SLP tokens as opposed to in BCH.
+
+
+## Specification
+
+### Extending Existing Payment Protocol
+
+This payment protocol is an extension of a portion of the [Simple Ledger Payment Protocol](https://github.com/simpleledger/slp-specifications/blob/master/slp-payment-protocol.md) and the [JSON Merchant Data](https://github.com/jeton-tech/payment-protocol-extensions/blob/master/json-merchant-data.md) specifications. Applications supporting these protocols are able, with minor modifications, to support the payment protocol described in this specification.
+
+
+### Postage Rate Request
+
+In order to determine the required postage rate for a given token, the sender's wallet queries the post office server. The server returns postage rate data which can then be used by the wallet to calculate the appropriate amount of SLP token value to include in an additional output as well as the proper payment address. Postage is paid in "stamp" units. Each stamp represents a certain number (typically 365) of satoshis that will be added to the transaction.
+
+#### REQUEST
+A GET request should be made to the Postage Protocol url.
+
+#### Headers
+* `Accept` should be set to `application/json`.
+
+#### Response
+The response will be a JSON format payload.
+
+#### Response Body
+* `version` - The version of the Postage Rate Request specification. This document defines version 1
+* `address` - The address to which postage payment must be made
```

Comment:
Does this change, and if so what causes this address to change in your current implementation?